server/docs/getting-started.md

```diff
@@ -142,384 +288,97 @@ async function initWeb3() {
+The `AppFactory` is how we create new virtual channels, and it needs to know:
+  * the address of the contract which provides the application logic (**HighRoller.sol**)
+  * the encodings for the application state and actions (to maintain the privacy of state and actions)
+  * which `Provider` can listen and propose updates to state.
 
+We'll input the contract address for **HighRoller.sol**, and fill in the details of the `Action` and `AppState` data types for the encodings.
 
 ```typescript
 async function install() {
   resetGameState();
+  
+  const contractAddress = '0x91907355C59BA005843E791c88aAB80b779446c9';
+  const actionEncoding = 'tuple(uint8 actionType, uint256 number, bytes32 actionHash)';
+  const stateEncoding = 'tuple(address[2] playerAddrs, uint8 stage, bytes32 salt, bytes32 commitHash, uint256 playerFirstNumber, uint256 playerSecondNumber)'
 
   let cfProvider = new cf.Provider(nodeProvider);
   let appFactory = new cf.AppFactory(contractAddress, {
-    actionEncoding: "tuple(uint8 actionType, uint256 number, bytes32 actionHash)",
-    stateEncoding: "tuple(address[2] playerAddrs, uint8 stage, bytes32 salt, bytes32 commitHash, uint256 playerFirstNumber, uint256 playerSecondNumber)"
+    actionEncoding,
+    stateEncoding
   }, cfProvider);
-
-  proposeInstall(appFactory);
-
-  cfProvider.on('installVirtual', onInstallEvent);
-  cfProvider.on('updateState', onUpdateEvent);
 }
 ```
 
-We can also describe the initialState, which consists of the two addresses and the Stage set to PRE_GAME:
+We’ll come back to define the function `resetGameState()` once we have a better sense of what that will entail.
 
-```typescript
-const initialState = {
-      playerAddrs: [
-        deriveAddress(
-          account.nodeAddress
-        ),
-        deriveAddress(
-          nodeAddress
-        )
-      ],
-      stage: HighRollerStage.PRE_GAME,
-      salt: HashZero,
-      commitHash: HashZero,
-      playerFirstNumber: 0,
-      playerSecondNumber: 0
-    };
-```
+### The AppFactory's proposeVirtualInstall Method
 
-In order to obtain the addresses, we include a couple of address-collecting utility functions
+The `install()` function will also call `proposeInstall(appFactory)`. This function is where we will implement `appFactory`'s `proposeInstallVirtual()` method, which will propose creating a virtual state channel with underlying application logic specified in `appFactory`'s instantiation. To propose the new virtual channel, we'll need to specify:
+  * initial state for the channel
+  * who is participating in the channel
+  * what are the stakes (and in what currency)
+  * how long before timeout
+  * the intermediary
+
+By looking at the application logic, we expect the initial game state to begin with the `playerAddrs` set and `state.STAGE` set to `Stage.PRE_GAME`. In order to obtain the addresses, we include a couple of address-collecting utility functions
```

Comment:
"begin with the `playerAddrs` set"... with the address of the user and the address of the counterparty (in this case, our hard-coded bot)